RELEASING.md

```diff
@@ -39,7 +39,10 @@ We use a tool called [github changelog generator](https://github.com/github-chan
     docker run -it --rm -v "$(pwd)":/usr/local/src/your-app githubchangeloggenerator/github-changelog-generator -u mongodb -p terraform-provider-mongodbatlas -t <GH_TOKEN> --breaking-labels "breaking-change" --enhancement-label "**Enhancements**" --bugs-label "**Bug Fixes**"  --issues-label "**Closed Issues**" --pr-label "**Internal Improvements**" --max-issues 1000
     ```
     To obtain your github personal access token you can use the following guide: [Authorizing a personal access token for use with SAML single sign-on](https://docs.github.com/en/enterprise-cloud@latest/authentication/authenticating-with-saml-single-sign-on/authorizing-a-personal-access-token-for-use-with-saml-single-sign-on)
- 
+
+#### Update Test Suite versions
+- Update [Test Suite workflow](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/.github/workflows/test-suite.yml) if needed. Check `provider_matrix` and `schedule_provider_matrix` to make sure that they include the latest patch of the latest 2 provider versions.
```

Comment:
I am not sure what the latest patch of the latest 2 provider versions means 🤔

Reply:
you're right it's confusing. what about this?  "... the latest patch version release of the latest 2 major versions"

at this moment, the 2 latest versions of the provider are 1.14.x and 1.13.x, and the latest patches make: 1.14.0, 1.13.1

do you have any suggestion?

Reply:
Maybe you can also provide an example

Reply:
finally we decided to use only the latest one at the moment